openqa-label-known-issues

```diff
@@ -175,7 +171,37 @@ investigate_issue() {
 }
 
 label_issue() {
-    local testurl="${1:?"Need 'testurl'"}"
+    local arg="${1:-}"
+    if [[ -z "$arg" ]]; then
+        echo "Usage: $0 <job_id> or https://<host>/tests/<job_id>" >&2
+        exit 1
+    fi
+
+    scheme="${scheme:-"https"}"
+    local job_id=""
+    local testurl=""
+
+    if [[ "$arg" =~ ^(https|http)?://([^/]+)/tests/([0-9]+)$ ]]; then
```

Comment:
The quotes on the left hand side of a `[[` condition don't make a difference:
```suggestion
    if [[ $arg =~ ^(https|http)?://([^/]+)/tests/([0-9]+)$ ]]; then
```
The same counts for the other `[[` conditions.